client/context/query.go

```diff
@@ -117,32 +115,35 @@ func (ctx CLIContext) GetAccountSequence(address []byte) (uint64, error) {
 // error is returned if it does not.
 func (ctx CLIContext) EnsureAccountExists() error {
 	addr := ctx.GetFromAddress()
-	accountBytes, err := ctx.QueryStore(auth.AddressStoreKey(addr), ctx.AccountStore)
-	if err != nil {
-		return err
-	}
-
-	if len(accountBytes) == 0 {
-		return ErrInvalidAccount(addr)
-	}
-
-	return nil
+	return ctx.EnsureAccountExistsFromAddr(addr)
 }
 
 // EnsureAccountExistsFromAddr ensures that an account exists for a given
 // address. Instead of using the context's from name, a direct address is
 // given. An error is returned if it does not.
 func (ctx CLIContext) EnsureAccountExistsFromAddr(addr sdk.AccAddress) error {
```

Comment:
I know you did not introduce this change, but any chance you can rename `EnsureAccountExistsFromAddr` => `EnsureAccountExists`. Idk why we added that suffix?

Reply:
It looks like there is still an `EnsureAccountExists` function that checks the `FromAddress` then defers to `EnsureAccountExistsAddr`

Reply:
Yeah, there is still EnsureAccountExists method.